<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoFormPageLzy</class>
 <widget class="QWidget" name="InfoFormPageLzy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="container" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="birthdayEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="locationEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>生日：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>所在地：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addFriendButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>添加好友</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendMessageButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="agreeButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>通过好友申请</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rejectButton">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>288</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>拒绝好友申请</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeFriendButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>删除好友</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
